src/screens/PluginStore/components/tests/PluginList.test.tsx

```
@@ -0,0 +1,218 @@
+          {plugin.description}
+        </div>
+        <div data-testid={`plugin-author-${plugin.id}`}>{plugin.author}</div>
+        <img
+          data-testid={`plugin-icon-${plugin.id}`}
+          src={plugin.icon}
+          alt="Plugin Icon"
+        />
+        <button
+          onClick={() => onManage(plugin)}
+          data-testid={`plugin-action-btn-${plugin.id}`}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)